.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish to PyPI
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+jobs:
+  publish-release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name:
```

Comment:
does this need a name?

Reply:
Oops. Step names aren't required, but omitting one here was not intentional.